test/M4/M4.py

```diff
@@ -60,23 +58,16 @@
 
 test.run()
 
-import sys
-
-if sys.platform == 'win32':
-    # Handle carriage returns.
-    test.must_match(test.workpath('aaa.x'), "line 1\r\nmym4.py\r\nline 3\r\n")
-else:
-    test.must_match(test.workpath('aaa.x'), "line 1\nmym4.py\nline 3\n")
-
-
+test.must_match(test.workpath('aaa.x'), "line 1" + os.linesep + "mym4.py" + os.linesep + "line 3" + os.linesep)
```

Comment:
os.linesep.join([....]) ?

Reply:
I'd tend to agree, "adding" multiple substrings is usually frowned upon, even though this is "only" a testcase....

Reply:
updated in dc4b8ba